biz.aQute.bndlib/src/aQute/bnd/osgi/Clazz.java

```diff
@@ -479,6 +480,21 @@ public TypeRef getType() {
 			return type;
 		}
 
+		public int getModuleAccess() {
```

Comment:
These methods in ClassDef shoud be package private to allow future freedom to refactor.

Reply:
right! will do